ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml

```diff
@@ -10,6 +10,7 @@
     scenario:
       - create
       - dmcrypt
+      - prepare_activate
```

Comment:
Where is this from?

Reply:
```
$ tox -l
centos8-filestore-create
centos8-filestore-prepare_activate
centos8-filestore-dmcrypt
centos8-bluestore-create
centos8-bluestore-prepare_activate
centos8-bluestore-dmcrypt
$ pwd
/home/fedora/ceph/src/ceph-volume/ceph_volume/tests/functional/lvm
```

Reply:
but
```
% l tests/functional/lvm/centos8/bluestore                                                                                              :(
total 0
drwxr-xr-x 1 jan jan 128 Feb 25 16:46 create
drwxr-xr-x 1 jan jan 128 Feb 25 16:46 dmcrypt
```
seems to be the tox.ini needs fixing.

Reply:
in any case, create is only a convenience command for prepare and activate.

Reply:
Okay, got rid of it.